src/internal_modules/roc_core/list.h

```diff
@@ -147,6 +147,36 @@ class List : public NonCopyable<> {
         insert_(element, NULL);
     }
 
+    //! Pop first element from list
```

Comment:
nitpick: Please add to then end of the first line, IIRC it's required by doxygen auto-brief mode.

```suggestion
    //! Pop first element from list.
```